config/dev.exs

```diff
@@ -1,11 +0,0 @@
-import Config
```

Comment:
@KronicDeth Oh, this is part of the "removed some dead code" change I mentioned in the description so not related to the Phoenix upgrade at all. This repo started as a project generated with `mix phx.new` but there's actually no need to run it as a standalone Phoenix server (it's intended to be used as a dependency within other Phoenix applications). Because of that, I decided to take the opportunity to clean up all of the auto-generated Phoenix stuff since library consumers won't need it 🙂